kernel/src/context.rs

```diff
@@ -29,6 +29,16 @@ impl Context<'_> {
     pub fn get_user_space(&self) -> CurrentUserSpace {
         CurrentUserSpace(self.task.user_space().unwrap().vm_space().clone())
     }
+
+    /// Sets thread-local storage pointer.
+    pub fn set_tls_pointer(&self, tls: usize) {
```

Comment:
I am doubtful about adding `set_tls_pointer` and `tls_pointer` methods for `Context`. They are rarely used. And even without them, the user can still use `ctx.task` to call its `set_tls_pointer` and `tls_pointer` methods directly.

Reply:
Right, it's redundant. I'll remove those methods.